packages/app/src/elements/field.ts

```diff
@@ -152,7 +152,7 @@ export class FieldElement extends LitElement {
 
     private _collapseSuggestions() {
         window.setTimeout(() => {
-            const drawer = this._valueInput.querySelector("pl-drawer") as Drawer;
+            const drawer = this._valueInput?.querySelector("pl-drawer") as Drawer;
```

Comment:
This was failing because the test ran too fast, and the timeout ran without a `this._valueInput` available.